files/grest/rpc/pool/pool_update.sql

```diff
@@ -0,0 +1,32 @@
+CREATE OR REPLACE FUNCTION grest.pool_update(_pool_bech32 text, _current_epoch_no numeric default 0, _state text default '' )
```

Comment:
We could potentially call this one pool_updates (plural)?

Reply:
👍